### PR TITLE
Adding and removing a prop on settlement entity - Fixed timestamps on pair totals entity that was wrong

### LINKED ISSUE #50: In pairs entity, firstTradeTimestamp prop is wrong.
Anxo noted firstTradeTimestamp is wrong. the time saved is not being rounded by hour or day.

### LINKED ISSUE #51: In settlement entity, firstTradeTimestamp doesn't make sense.
In settlement entity there's a firstTradeTimestamp that doesn't make sense. One settlement has a bunch of trades. Is one tx which executes a lot of trades. So they will naturally have all the same date.

In the other hand it'll be great to add blockNumber in the settlement entity.

### PR DESCRIPTION
**Pull request by GabrielCamba**
Fixed value on daily and hourly pairs of timestamp that was reported wrong. The timestamp was wrong because on pairDailyTotal, it was using directly the timestamp from the block instead of using the conversion for the daily infomation. 
We changed that for pairDailyTotals entity and pairHourlyTotals entity because it was wrong in both of them.

On Settlement entitiy there was created a new prop called blockNumber to show the block where the settle was created. It was removed the firstTradeTimestamp because it's a prop that belongs to a trade and can be consumed form there.

Closes #50 
Closes #51

Deployed and indexing here:
https://thegraph.com/hosted-service/subgraph/gabrielcamba/cow-two

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -115,8 +115,8 @@ type Settlement @entity {
   id: ID!
   "Transaction hash"
   txHash: Bytes!
-  "First trade timestamp"
-  firstTradeTimestamp: Int!
```
Comment (alfetopito) [inline schema.graphql:119]: I thought we would use that in the UI.
Have we agreed to remove it?

**Comment by anxolin** [comment]
"Minor fixes"
as a title is not good

> Fixed value on daily and hourly pairs of firstTradeTimestamp that was reported wrong.

Was wrong in what sense? Can't you give more context of what the issue was and what's your solution.

> Removed timestamp form Settlements entity

Maybe a few words why helps a random reader, and documents the changes you apply to the code.

> Added blockNumber to Settlements entity

I guess this is self explanatory

File: schema.graphql (hunk 1 above)
Comment (GabrielCamba) [reply schema.graphql:119]: I'm pretty sure we agreed on removing it. But please let me know if you want to keep it.

Comment (GabrielCamba) [reply schema.graphql:119]: Discussed with Rami, Need to keep this prop

**Comment by alfetopito** [comment]
Can you provide a query to test the indexed changes?

**Comment by GabrielCamba** [comment]
> Can you provide a query to test the indexed changes?

you can try using this:

``` GraphQL
{
  settlements (first: 5) {
    blockNumber
  }
  pairDailies (first: 5) {
    timestamp
  }
  pairHourlies (first: 5) {
    timestamp
  }
}
```
- The blockNumber property wasn't there in the previous version
- regarding the timestamp in dailies the timestamp should reference one per day at 0:00:00 GMT. Use this for checking: https://www.unixtimestamp.com/
- regarding the timestamp in hourlies, should be x:00:00 where x is one single hour. 

Please notice I redeployed the [version](https://thegraph.com/hosted-service/subgraph/gabrielcamba/cow-two?version=pending), that's because I replaced to do a different test.

**Review by alfetopito — APPROVED** [review]
Checked and I got the results as instructed 👍

**Pull request merged**